handbook/engineering/README.md

```diff
@@ -84,6 +86,61 @@ Everyone at the company is welcome to attend. All engineers are asked to attend.
 - Social
   - Structured and/or unstructured social activities
 
+### User story discovery
+
+User story discovery meetings are scheduled as needed to align on large or complicated user stories. Before a discovery meeting is scheduled, the user story must be prioritized for product drafting and go through the design and specification process. When the user story is ready to be estimated, a user story discovery meeting may be scheduled to provide more dedicated, synchronous time for the team to discuss the user story than is available during weekly estimation sessions.
+
+All participants are expected to review the user story and associated designs and specifications before the discovery meeting.
+
+#### Participants
+
+- Product Manager
+- Product Designer
+- Engineering Manager
+- Backend Software Engineer
+- Frontend Software Engineer
+- Product Quality Specialist
+
+#### Agenda
+
+- Product Manager: Why this story has been prioritized
+- Product Designer: Walk through user journey wireframes
+- Engineering Manager: Review specifications and any defined sub-tasks
+- Software Engineers: Clarifying questions and implementation details
+- Product Quality Specialist: Testing plan
+
+#### Participants
+
+Everyone at the company is welcome to attend. All engineers are asked to attend. The subject matter is focused on engineering.
+
+#### Agenda
+
+- Announcements
+- Engineering KPIs review
+- “Tech talks”
+  - At least one engineer from each product group demos or discusses a technical aspect of their recent work.
+  - Everyone is welcome to present on a technical topic. Add your name and tech talk subject in the agenda doc included in the Eng Together calendar event.
+- Social
+  - Structured and/or unstructured social activities
+
```

Comment:
@sabrinabuckets is correct, this section should be removed as it's a duplication.